server/cluster/coordinator.go

```diff
@@ -916,8 +940,20 @@ func (s *scheduleController) GetInterval() time.Duration {
 }
 
 // AllowSchedule returns if a scheduler is allowed to schedule.
-func (s *scheduleController) AllowSchedule() bool {
-	return s.Scheduler.IsScheduleAllowed(s.cluster) && !s.IsPaused() && !s.cluster.GetUnsafeRecoveryController().IsRunning()
+func (s *scheduleController) AllowSchedule(diagnosable bool) bool {
+	if !s.Scheduler.IsScheduleAllowed(s.cluster) {
+		if diagnosable {
+			s.diagnosticWorker.setResultFromStatus(scheduling)
```

Comment:
Does scheduling mean that it can generate the operator instead of less than the limit?

Reply:
We need to modify the definition of scheduling which means means the current scheduler is generating.